src/types/interfaces.ts

```diff
@@ -864,3 +864,38 @@ export interface ISdkMediaState {
   /** if permissions have been requested by the sdk */
   cameraPermissionsRequested: boolean;
 }
+
+export interface ISdkSoftphoneSessionRequest extends IStartSessionParams {
```

Comment:
You will need to add this to the docs as well. I think `doc/softphone.md` will be best. 

I think the `softphone.md` will need a little face lift to make it conform to the other markdown file formats (see video.md for the simplest format). I admit I didn't really change this during my doc re-write because there wasn't any softphone specific functionality. Now that there is, it will be nice to have the same format at the others.

Reply:
Function changes have been pushed. Will update the docs in the morning!

Reply:
Docs are updated btw.